_posts/2020-07-29-parallelized-sampling/parallelized-sampling.Rmd

```diff
@@ -0,0 +1,290 @@
+---
+title: "Parallelized sampling using exponential variates"
+description: > 
+  How can the seemingly iterative process of weighted sampling without replacement be transformed into something highly parallelizable? Turns out a well-known technique based on exponential variates accomplishes exactly that.
+author:
+  - name: Yitao Li
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+bibliography: bibliography.bib
+date: 07-29-2020
+categories:
+  - Concepts
+  - Distributed Computing
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/dice.jpg
+header-includes:
+   \usepackage{algorithm2e}
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+As part of our recent work to support weighted sampling of Spark dataframes in `sparklyr`, we embarked on a journey searching for algorithms that can perform weighted sampling, especially sampling without replacement, in efficient and scalable ways within a distributed cluster-computing framework such as Apache Spark.
```

Comment:
I can't remember what you've done for past posts. so just double check that you have been doing dataframes as one word rather than two for consistency.